### ISSUE TITLE
Assignment 2 review

### ISSUE THREAD

**Issue by ycui0008** [etc5521-2020/assignment-1-koala#9]
1. The GIF at the start of the report is fancy, I like it.
2. It is great that most of the plots are interactive. But the labels of some plots are too small to be seen because of too many countries. For example, figure 3.1 and 3.3. And figure 3.2 may be better if coord_flip() is used.
3. References are clear and in the right format. But it can be better if the packages used in the report is cited correctly.

In conclusion, this report is funny to read and very informative. It relates to the current hot topic -- energy saving and new energy. The report has a comprehensive CONCLUSION, which is very good.
